src/components/common/footer/Footer.js

```diff
@@ -1,24 +1,37 @@
 import React from 'react';
 import styled from 'styled-components';
 
+import { ReactComponent as Logo } from '../../../assets/logo-yellow.svg';
+import { ReactComponent as InstagramIcon } from '../../../assets/socialmedia/instagram.svg';
+import { ReactComponent as TwitterIcon } from '../../../assets/socialmedia/twitter.svg';
+import { ReactComponent as LinkedInIcon } from '../../../assets/socialmedia/linkedin.svg';
+
 import { colours, spacing } from '../../../theme';
 
 const FooterElem = styled.footer`
   background: ${colours.black};
-  height: 35vh;
+  display: flex;
+  justify-content: space-between;
+  padding: ${spacing.medium};
+  position: relative;
 `;
 
-// Dont forget to fix footer to bottom when rest of content is on page (Anna B and Burhan)
-
-const Email = styled.h4`
-  color: ${colours.white};
-  padding: ${spacing.medium};
+const IconContainer = styled.div`
+  display: flex;
+  min-width: 20vw;
+  justify-content: space-between;
+  
 `;
 
 export const Footer = props => {
```

Comment:
no need for props on line 26, as it is never used.
```suggestion
export const Footer = () => {
```